src/tensor/array.cr

```diff
@@ -0,0 +1,7 @@
+require "./tensor"
+
+class Array(T)
```

Comment:
Can you move this to `src/extensions`, to be consistent with the number extensions?

Reply:
Absolutely! Should I do that, and just make another commit?